data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2378 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# Transports
+
+ship "Koryfi"
+	sprite "ship/avgi koryfi/koryfi"
+		"frame rate" 30
+	thumbnail "thumbnail/avgi koryfi"
+	attributes
+		category "Transport"
+		"cost" 1860000
+		"shields" 600
+		"hull" 1800
+		"required crew" 2
+		"bunks" 24
+		"mass" 180
```

Comment:
Seems to reach impressive speeds in stock loadout- 783 max, 272 acceleration fully loaded, 136 turn fully loaded. Still slow compared to the very, very fast Avgi warships; but in general mass and drag for Avgi ships seem quite low. I imagine this is at least partly intentional, but given Magnetoplasma Drives seem to be about on par with Remnant engines of similar size and similar fast human ships (the Blackbird, for one) reach about half the speed on the same propulsion I suspect it needs something of a nerf. Have the Avgi ships been hit by the mass rebalance yet? That might be part of the problem.